en/docs/reference/connectors/salesforce-marketing-cloud-connector/salesforce-marketing-cloud-email-prerequisites.md

```diff
@@ -0,0 +1,99 @@
+# Prerequisites for Sending an Email
+
+**Note**: Make sure authentication is set up before sending emails to prevent them from being bounced. Specifically, if the sending domain fails any of the SPF, DKIM, or DMARC checks, the email will be rejected.
```

Comment:
```suggestion
!!!Note
    Make sure authentication is set up before sending emails to prevent them from being bounced. Specifically, if the sending domain fails any of the SPF, DKIM, or DMARC checks, the email will be rejected.
```